telegram/bot.py

```diff
@@ -3384,9 +3419,17 @@ def send_poll(self,
         data = {
             'chat_id': chat_id,
             'question': question,
-            'options': options
+            'options': options,
+            'is_anonymous': is_anonymous,
```

Comment:
If I see correctly we currently only mandatory arguments are added to `data` by default in the other methods. For optional args its always `if arg: data['arg'] = arg`. Is there a special reason you did this differently here?

Reply:
nope, me not being fully there probably. thanks.